templates/tfengine/components/cicd/terraform.tfvars

```diff
@@ -69,20 +69,29 @@ envs = [
           {{- if has .triggers "validate"}}
           run_on_push = {{get .triggers.validate "run_on_push" true}}
           run_on_schedule = "{{get .triggers.validate "run_on_schedule" ""}}"
+          {{- else}}
+          run_on_push = false
+          run_on_schedule = ""
           {{- end}}
         }
         plan = {
           skip = {{not (has .triggers "plan")}}
           {{- if has .triggers "plan"}}
           run_on_push = {{get .triggers.plan "run_on_push" true}}
           run_on_schedule = "{{get .triggers.plan "run_on_schedule" ""}}"
+          {{- else}}
+          run_on_push = false
+          run_on_schedule = ""
```

Comment:
Actually i think you can probably do this so it's cleaner. could you try it?

```
          run_on_push = {{get .triggers "plan.run_on_push" true}}
          run_on_schedule = "{{get .triggers "plan.run_on_schedule" ""}}"
```

Reply:
It worked. I ran `regen` and the output didn't changed.

Thanks